faq/my-test-does-not-show-up-in-the-deliver-administration-part.md

```diff
@@ -8,4 +8,4 @@ created_at: 2016-03-03
 There are different reasons for that, please check all the common issues:
 
 - Have you create a delivery related to this test? 
-- Have you associated your delivery with the group and considering that the test taker you a re connected with shall eb a member of?
\ No newline at end of file
+- Have you associated your delivery with the group and considering that the test-taker you a re connected with shall eb a member of?
```

Comment:
This file looks like it needs some more corrections.

Reply:
We have an agreement that we spell Test-taker always starting with a capital _T_. Quite possibly we never told you that, but now I did :-)